yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -110,6 +111,14 @@ def test_unit_cosmo():
         expected_time = 3.756241729312462e+17 # in seconds
         assert_equal(ds.current_time.in_units('s').value, expected_time)
 
+    assert ('io', 'particle_formation_time') in ds.field_list
```

Comment:
Maybe we could source-out the part testing for the different times in a new test, and test it for cosmological (`ramsesCosmo`), non cosmological (`ramsesNonCosmological`), as well as outputs in the new format (`ramses_new_format`).